### ISSUE TITLE
[BUG] chromebook keyboard controls stoped working

### ISSUE THREAD

**Issue by kingsmen1234** [PojavLauncherTeam/PojavLauncher#3879]
### Describe the bug

my keyboard controls had stoped working on my game after the most recent pojav update and changed my gui

### The log file and images/videos

[latestlog.txt](https://github.com/PojavLauncherTeam/PojavLauncher/files/10508491/latestlog.txt)

### Steps To Reproduce

```markdown
update pojave launcher from the last stable version to the most recent
```

### Expected Behavior

i expected to be able to use my mouse and keyboard to play on my chromebook pojav launcher but it is acting as if it were a mobile device

### Platform

```markdown
- Device model: hp lenvo chromebook 32GB
- CPU architecture: 64 bit
- Android version: Version 109.0.5414.94 (Official Build) (64-bit)
- PojavLauncher version: [Latest Release || version 3.3.1.1_rel_20210206 ]
```

**Comment by GaucovyAero** [comment]
wdym by it acting as a mobile device?

**Comment by Mathias-Boulay** [comment]
Can you state your EXACT pojav version ?
Without such knowledge, we can't help you

**Comment by khanhduytran0** [comment]
> Can you state your EXACT pojav version ?

It is stated in latestlog.txt, by verifying game launch time. However, it is definitely not the latest build.

> Info: Launcher version: dahlia-218-6888cb7d8-v3_openjdk
> ...
> ---- Minecraft Crash Report ----
> Time: 1/26/23, 2:33 AM

**Comment by Mathias-Boulay** [comment]
> > Can you state your EXACT pojav version ?

Fair enough, turns out I just read what the user sent, not the log itself

**Comment by Mathias-Boulay** [comment]
@kingsmen1234 Try out the current dev build instead, controls should work as expected

**Comment by speedythehedghog** [comment]
how do i get that

**Comment by speedythehedghog** [comment]
> @kingsmen1234 Try out the current dev build instead, controls should work as expected

where do i find and download the apk for the dev build?

**Comment by Mathias-Boulay** [comment]
In the actions tab

**Comment by kingsmen1234** [comment]
> > Can you state your EXACT pojav version ?
> 
> It is stated in latestlog.txt, by verifying game launch time. However, it is definitely not the latest build.
> 
> > Info: Launcher version: dahlia-218-6888cb7d8-v3_openjdk
> > ...
> > ---- Minecraft Crash Report ----
> > Time: 1/26/23, 2:33 AM

im sorry its been a hard few days since this was posted but my veiwer only starts showing the text of the .txt file after the "> > ---- Minecraft Crash Report ----" part of the log

**Comment by kingsmen1234** [comment]
> In the actions tab

also how do i use said apk to update the app

**Comment by Mathias-Boulay** [comment]
well
the apk is inside the zip

**Comment by kingsmen1234** [comment]
> well the apk is inside the zip

sorry i forgot to tell you it has a parsing problem even with the correct arm and hardware requirements

**Comment by GaucovyAero** [comment]
> > well the apk is inside the zip
> 
> sorry i forgot to tell you it has a parsing problem even with the correct arm and hardware requirements

make sure you dont use chrome as your browser to download zips and jars, its gonna corrupt those files
